rep-2005.rst

```diff
@@ -39,20 +39,53 @@ The list below is curated to include repositories that are:
 
 We aim that all of these repositories are also:
 
-* actively maintained and following accepted software development processes, including code review and testing.
+* actively maintained and following accepted software development processes, including code review and testing; and
+* at Quality Level 3 or better, as defined in REP 2004; and
+* are maintained by an organization or at least two individuals.
+
+The list should be self-contained and exhaustive: if a package appears on the list, then all of its ROS 2 dependencies must also appear on the list (third-party or system dependencies should not be included).
 
 
 Change process
 ==============
 
 This content is maintained at GitHub in the [REP repo](https://github.com/ros-infrastructure/rep).
-To propose changes, such as adding or removing items, make a pull request to that repo.
-The ensuing public review and discussion will evaluate the change in light of the purpose and scope explained above, with the final merge decision being made by one of the REP maintainers.
+To propose changes, such as adding or removing items, make a pull request (PR) to that repo.
+The ensuing public review and discussion will evaluate the change in light of the purpose and scope explained above.
+The final merge decision on a given PR will be made by the ROS 2 Technical Steering Committee (TSC) by a simple voting process:
+
+* Each TSC member organization may vote on a PR by commenting with a score according to the [REP voting guidelines](https://www.ros.org/reps/rep-0010.html#voting-scores).
```

Comment:
Link does not render correctly in Github, in the rest of this file links are expressed in the form `` `ament/ament_cmake <https://github.com/ament/ament_cmake>` ``

Reply:
Thanks for catching that; 9734990.